robot/Cumulus/tests/browser/gift_entry/reorder_template.robot

```diff
@@ -0,0 +1,109 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             API Check And Enable Gift Entry
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Keywords ***
+
+Setup Test Data
+    [Documentation]     Setsup namespace prefix
+    ${ns} =  Get NPSP Namespace Prefix
+    Set suite variable    ${ns}
+
+Get Template Builder Field Names
+  @{BUILDER_FORM_FIELDS} =  Return Template Builder Titles  template_builder_fields
+  ${BUILDER_FIELD_LABELS} =  Create List
+
+  FOR  ${LABEL}  IN  @{BUILDER_FORM_FIELDS}
+      ${NAME} =  Get Text  ${LABEL}
+      Append to List  ${BUILDER_FIELD_LABELS}  ${name}
+  END
+  Set Suite Variable  ${BUILDER_FIELD_LABELS}
+
+Get Template Builder Section Names
+  @{BUILDER_SECTION_TITLES} =  Return Template Builder Titles  template_builder_sections
+  ${BUILDER_S_TITLES} =  Create List
+
+  FOR  ${LABEL}  IN  @{BUILDER_SECTION_TITLES}
+      ${NAME} =  Get Text  ${LABEL}
+      Append to List  ${BUILDER_S_TITLES}  ${NAME}
+  END
+  Set Suite Variable  ${BUILDER_S_TITLES}
+
+Get Gift Entry Form Field Names
+  @{GIFT_FORM_FIELDS} =  Return Gift Form Titles  gift_entry_form_fields
+  ${FORM_FIELD_LABELS} =  Create List
+
+  FOR  ${LABEL}  IN  @{GIFT_FORM_FIELDS}
+      ${NAME} =  Get Text  ${LABEL}
+      Append to List  ${FORM_FIELD_LABELS}  ${NAME}
+  END
+  Set Suite Variable  ${FORM_FIELD_LABELS}
+
+Get Gift Entry Form Section Names
+  @{FORM_SECTION_TITLES} =  Return Gift Form Titles  gift_entry_form_sections
+  ${FORM_S_TITLES} =  Create List
+
+  FOR  ${LABEL}  IN  @{FORM_SECTION_TITLES}
+      ${NAME} =  Get Text  ${LABEL}
+      Append to List  ${FORM_S_TITLES}  ${NAME}
+  END
+  Set Suite Variable  ${FORM_S_TITLES}
+
+*** Test Cases ***
+
+Reorder and Modify GE Template Fields
+  [Documentation]                       Tests adding, deleting, and reordering form fields on the template builder,
+  ...                                   and compares the order of the template form fields with the order of the 
+  ...                                   gift form in a batch created from that template.
+  [tags]                                unstable                    feature:GE          W-039563
+  ${TEMPLATE} =                         Generate Random String
```

Comment:
use lower case for template as it is a local variable

Reply:
Fixed